### ISSUE TITLE
Memory Tiering: Validation on CRI Tools tests

### ISSUE THREAD

**Issue by msivosuo** [intel/cri-resource-manager#240]
Memory tier policy validation on CRI Tools tests

-Run CRI Tools tests and check we are not breaking anything (compare with and without CRI-RM).

Target: 2020-05-08

**Comment by askervin** [comment]
Used `critest` for validating CRI-RM conformance with policies:
- null
- topology-aware
- memtier
- no CRI-RM, direct connection to `containerd`.

Code and instructions for rerunning the tests are in my [critest branch](https://github.com/askervin/cri-resource-manager/commits/56S_critest), [tests/critest](https://github.com/askervin/cri-resource-manager/tree/56S_critest/test/critest) directory.

Tests detected a crash in memtier and topology-aware policies. The crash is fixed in PR #266.

`critest` summary with the fix is the same for all policies, and for direct `containerd` connection without CRI-RM running in-between:
```
Summarizing 2 Failures:

[Fail] [k8s.io] AppArmor runtime should support apparmor [It] should fail with with an unloaded profile
/home/ask/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/apparmor_linux.go:131

[Fail] [k8s.io] Networking runtime should support networking [It] runtime should support port mapping with host port and container port [Conformance]
/home/ask/go/src/github.com/kubernetes-sigs/cri-tools/pkg/validate/networking.go:253

Ran 74 of 81 Specs in 218.998 seconds
FAIL! -- 72 Passed | 2 Failed | 0 Pending | 7 Skipped
```

**Comment by askervin** [comment]
Implementing test environment that enables running `critest` with minimal host requirements (dependencies, configuration) is future work, tracked in issue #123.
